### ISSUE TITLE
syphon.root() doesn't render

### ISSUE THREAD

**Issue by Janiczek** [scttnlsn/syphon#2]
In my app syphon never gets to call the `context.render()` function. I have a blank page instead of my component. **(Eg. the example in `README` doesn't work.)**

I'm calling:

``` javascript
syphon.root(App, state, {
  dispatcher: dispatcher,
  el: document.getElementById('app')
});
```

which calls `context.mount` which is currently defined as:

``` javascript
Context.prototype.mount = function (el) {
  var self = this;

  this.state.addWatch('render', function () {
    self.render(el);
  });
};
```

If I modify the `context.render()` function to `console.log("something")`, it never fires.

**I have fixed it by adding another `self.render(el);` just before the `this.state.addWatch(...)` in `context.mount()` function - then everything works as expected.**

My env: syphon `0.1.2`, react `0.12.2`, reactify `1.0.0`

**Comment by scttnlsn** [comment]
Thanks for pointing this out.  I've been using a router that updates the state immediately after loading the page and never noticed this.  I think a `this.render(el)` in `mount` makes sense.

**Comment by scttnlsn** [comment]
- Fixed in https://github.com/scttnlsn/syphon/commit/bf07117ba5dcebff1e9929ddbd3753ae350577d8
- Released in v0.1.3

**Comment by Janiczek** [comment]
Awesome, thanks!
